packages/devtools_app/lib/src/shared/editor/editor_client.dart

```diff
@@ -262,18 +260,40 @@ class EditorClient extends DisposableController
   }) async {
     final method = editableArgumentsMethodName.value;
     if (method == null) return null;
-    final response = await _callLspApi(
-      method,
-      params: {
-        'type': 'Object', // This is required by DTD.
-        'textDocument': textDocument.toJson(),
-        'position': position.toJson(),
-      },
-    );
-    final result = response.result[Field.result];
-    return result != null
-        ? EditableArgumentsResult.fromJson(result as Map<String, Object?>)
-        : null;
+    try {
+      final response = await _callLspApi(
+        method,
+        params: {
+          'type': 'Object', // This is required by DTD.
+          'textDocument': textDocument.toJson(),
+          'position': position.toJson(),
+        },
+      );
+      final result = response.result[Field.result];
+      return result != null
+          ? EditableArgumentsResult.fromJson(result as Map<String, Object?>)
+          : null;
+    } on RpcException catch (e, st) {
+      // We expect content modified errors if a user edits their code before the
+      // request completes. Therefore it is safe to ignore.
+      if (e.code != AnalysisServerError.contentModifiedError.code) {
+        final errorMessage = e.message;
+        reportError(
+          errorMessage,
+          errorType: PropertyEditorSidebar.getEditableArgumentsIdentifier,
+          stack: st,
+        );
+      }
+      return null;
+    } catch (e, st) {
+      final errorMessage = 'Unknown error: $e';
+      reportError(
+        errorMessage,
+        errorType: PropertyEditorSidebar.getEditableArgumentsIdentifier,
+        stack: st,
+      );
```

Comment:
nit: we could reduce duplication by putting this in a `finally` clause. If we had a variable `String? errorMessage` defined before the try, the `on RpcException` and `catch` blocks could just set the `errorMessage` variable that is used in the `finally` block that calls `reportError`

Reply:
Done!